nuitka/distutils/DistutilCommands.py

```diff
@@ -231,6 +232,18 @@ def _build(self, build_lib):
                     if fullpath.lower().endswith((".py", ".pyw", ".pyc", ".pyo")):
                         os.unlink(fullpath)
 
+            # If the Python module has more than one parent package (e.g.
+            # 'a.b.mod'), the compiled module will be in 'a.b/mod.so'. Move it
+            # to 'a/b/mod.so', to make imports work.
+            if package and "." in package:
+                compiled_package_path = os.path.join(build_lib, package)
+                assert os.path.isdir(compiled_package_path), compiled_package_path
+
+                parts = package.split(".")
+                fixed_package_path = os.path.join(build_lib, *parts)
+                dir_util.copy_tree(compiled_package_path, fixed_package_path)
```

Comment:
We have `nuitka.utils.FileOperation.copyTree` for that. Arguably importing distutils cannot be all that wrong in this module, but still.